### DISCUSSION TITLE
Displayed value varies between cards

### DISCUSSION THREAD

**Discussion by allseven** [ulic75/power-flow-card#127] [Q&A]
I have an Entity Card which displays the value 'myenergi_zappihub_home_consumption' correctly (869W at the moment).

Using the same sensor in the power-flow-card the value displayed at the same time is 1.1kW and the difference of 0.23kW remains as the consumption varies.

Am I missing something in the configuration?

**Comment by Jay66Bee** [comment]
Same here. 

Power flow card shows 5.3KW solar, 4.1KW home, 2.2KW battery charging & 1.2KW battery discharging

HA Energy page shows 4.6KW solar, 3.7KW home, 1.8KW battery charging & 1.0KW battery discharging

Only grid import/export is correct.

Used the same entities for both.

**Reply by kpfleming** [reply]
Those two cards do not show the same thing. The power-flow card shows instantaneous readings as they come in, it does not show summaries or historical data. The energy card shows updated-once-per-hour summaries. You can't use the same entities for both, because the power-flow card needs *power* entities (W/kW) and the energy card needs *energy* entities (Wh, kWh).
